code/modules/antagonists/thief/thief_objectives.dm

```diff
@@ -0,0 +1,36 @@
+
+GLOBAL_LIST_INIT(hoarder_targets, list(
+	/obj/item/clothing/gloves/color/yellow,
+	/obj/item/mod/core
+))
```

Comment:
This needs to be expanded with valuable items that the station will run out of if someone is actively going around collecting them. I'm taking suggestions!

Reply:
adding stunbatons

Reply:
Extended capacity oxygen tanks and medkits come to mind

Reply:
Bounty objectives that cant be crafted, like briefcases, donk pockets, and crayons